doc/changes/changes_0.8.0.md

```diff
@@ -0,0 +1,9 @@
+# Exasol Release Droid 0.8.0, released 2021-??-??
```

Comment:
I think it's already in production. So I would suggest to change version to 1.0.0

Reply:
but you can do it in a upcoming PR or release

Reply:
I keep it with 0 because I don't have integration tests and it's still a lot of bugs reported. For example, your bug when the workflows are not running. I would like to release 1.0.0 when it's stable